sdk/python/foundation-models/system/inference/text-to-image/safe-text-to-image-inpainting-batch-deployment.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
Try opening this notebook in "Preview" mode on git, it throws some error. I think some jupyternotebook variable is missing